app/model/frontsapi.scala

```diff
@@ -126,11 +130,27 @@ trait UpdateActionsTrait {
       case Success(_) => collectionJson
     }
 
+  private def v2ArchiveBlock(id: String, collectionJson: CollectionJson, identity: User): CollectionJson =
+    Try(faciaApiIO.v2Archive(id, collectionJson, identity)) match {
+      case Failure(t: Throwable) => {
+        structuredLogger.putLog(LogUpdate(ArchiveUpdate(id), "faciaTool"), "error", Some(new Exception(t)))
+        collectionJson
+      }
+      case Success(_) => collectionJson
+    }
+
   def putMasterConfig(config: ConfigJson, identity: User): Option[ConfigJson] = {
     faciaApiIO.archiveMasterConfig(config, identity)
     faciaApiIO.putMasterConfig(config)
   }
 
+  def v2UpdateCollection(id: String, collection: CollectionJson, identity: User): CollectionJson = {
+    val collectionWithGroupsRemoved = removeGroupIfNoLongerGrouped(id, collection)
+    val prunedCollection = pruneBlock(collectionWithGroupsRemoved)
```

Comment:
What does this line actually prune in reality? What sort of things are removed (or more to the point, what might we put here on the client that isn't wanted)?

Reply:
This removes empty metadata objects and removes empty  group objects if the collection is no longer grouped. The only thing I could think of why a collection is not grouped anymore is if someone changes the layout. These transformations are not really changing anything in the collection, just cleaning up empty stuff.

Reply:
👍 makes perfect sense